### ISSUE TITLE
Lambda errors with "no space left on device"

### ISSUE THREAD

**Issue by joehoyle** [humanmade/linter-bot#158]
For example, https://github.com/humanmade/altis-enhanced-search/pull/288/checks?check_run_id=3609500745 failed with the errors:

```
2021-09-15T11:31:47.694Z	2ed893a5-d030-4467-bc26-4cf34d666c7e	ERROR	Uncaught Exception 	{
    "errorType": "Error",
    "errorMessage": "ENOSPC: no space left on device, write",
    "code": "ENOSPC",
    "errno": -28,
    "syscall": "write",
    "stack": [
        "Error: ENOSPC: no space left on device, write"
    ]
}
```

(link to AWS console) https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:log-groups/log-group/$252Faws$252Flambda$252Fhm-linter/log-events/2021$252F09$252F15$252F$255B$2524LATEST$255Dccf83dd1193944c4864792c42401687d$3FfilterPattern$3D$26start$3D1631703600000$26end$3D1631707200000

I think this is because `/tmp` or whatever gets filled up, as we never clean up unpacked repositories perhaps? So the shared lambda container between jobs gets filled up (shared state issue).

I think we could either use the new Filesystems feature of Lambda, or we could stop "caching" the unpacked repos per run.

**Comment by joehoyle** [comment]
Hopefully this will be fixed in https://github.com/humanmade/linter-bot/pull/157. @rmccue mentioned @kadamwhite would be rolling it out to hm-linter once it's merged.

**Comment by rmccue** [comment]
Think this has been fixed by #157.
